src/tools/perf/cuda/cuda_alloc.c

```diff
@@ -69,15 +71,31 @@ uct_perf_cuda_alloc_reg_mem(const ucx_perf_context_t *perf,
                             unsigned flags,
                             uct_allocated_memory_t *alloc_mem)
 {
+    uct_md_attr_v2_t md_attr = { .field_mask = UINT64_MAX };
```

Comment:
since this is non-test code, maybe we want to use only `UCT_MD_ATTR_FIELD_REG_ALIGNMENT`.

Reply:
Thanks, makes sense